### ISSUE TITLE
new-file compound paths

### ISSUE THREAD

**Issue by dsmmcken** [microsoft/vscode-codicons#47]
Note: This isn't technically an issue with this icon set as published. Consider this a feature request, that would be appreciated.

I am running my own build step on the svg files, and packaging it with a slightly different process. The only icon that fails to build correctly for me is the new-file icon.

It's the only icon published as two separate paths rather than single compound path like the rest of the set. Can this be fixed?

I assume you might have master file you publish out of, or would you accept a PR that fixes it?

**Comment by misolori** [comment]
Curious where the icons are being used? Should be no problem to combine the paths and yes it's easier for me to do it on my end as the originals are saved in our Figma file. Can do that shortly.

**Comment by dsmmcken** [comment]
Thanks for the fast fix!

I am using the set in a browser app that uses the monaco-editor. I am subsetting the icons to just the ones our app + monaco-editor actually use, but also extending the set with additional icons specific to our app that visually match if they are appearing in the context with the monaco-editor.

monaco-editor currently ships all the icons to the browser, which isn't ideal.

**Comment by misolori** [comment]
Very cool!

> monaco-editor currently ships all the icons to the browser, which isn't ideal.

what are the issues with this, besides having all of the icons shipped?

**Comment by dsmmcken** [comment]
Just the bigger file size for the user to download, but I am not really complaining. It would be splitting hairs when the editor itself is the bulk of the download.

39kB gzipped for all the icons vs 633kB for just the main js. And monaco-editor uses  maybe a 1/3 of the full set? So the savings we are talking about might only amount to 20-30kB to the user. Unless the monaco-editor was really trying to slim down, it's probably not worth the separate effort to maintain a codicon.tff just for monaco.

**Comment by dsmmcken** [comment]
Hey -- I actually came across a few more icons that aren't a single compound path, like the debug, and stop-circle and a few others.

I've decided it is easiest on my end to just run my own svgo pass against the icons as they come from this repo, an assume they aren't always a single compound path. If anyone is looking for an easy way to have all the svg paths merged for you, the `force: true` option for `svgo` `mergePaths` seems to do the trick.
  
  ```yml
  - mergePaths:
      force: true # force paths to be joined, works cleanly so long as shapes don't intersect
  ```
